test/__tests__/data/facebook_pixel_proxy_output.json

```diff
@@ -85,7 +85,7 @@
   {
     "output": {
       "status": 400,
```

Comment:
For all fb pixel error responses, do we know how many changing attributes can come apart from `Object ID` ?
If they are handful, having common utility per destType to find those via regex & replacing with common value (xxxxxx) will work ?

Reply:
regex is additional processing, since they're handful we might as well have a value hard-coded in source-code like a constant, make use of it. It would convey the message perfectly and also would not cause any performance issues

Reply:
This is getting handled in server already. Hence resolving this comment